src/Frontend/Page/Admin.hs

```diff
@@ -592,7 +609,13 @@ instance Csv.FromRecord CsvUserRecord where
             | otherwise        = Just . UserLogin $ v !! i
 
 instance Csv.ToRecord CsvUserRecord where
-    toRecord (CsvUserRecord fn ln em li pw) = Csv.toRecord (show fn, show ln, show em, show li, show pw)
+    toRecord (CsvUserRecord fn ln em li pw) = Csv.toRecord
+        (          _fromUserFirstName            fn
+        ,          _fromUserLastName             ln
+        , maybe "" (show . internalEmailAddress) em
+        , maybe "" _fromUserLogin                li
+        , maybe "" id                            pw
```

Comment:
`pw ^. _Just`

This works because when `^.` is applied to a `Prism` it goes down to a `Fold` and get this type: `(^.) :: Monoid a => s -> Fold s a -> a`. In our case `(^.) :: Maybe ST -> Fold (Maybe ST) ST -> ST` which means that the monoid instance for ST is doing the work of defaulting to the empty string.